src/main/java/ToDoTask.java

```diff
@@ -0,0 +1,73 @@
+public class ToDoTask extends Task {
+  /**
+   * Constructor for ToDo task.
+   *
+   * @param description Description of task
+   */
+  public ToDoTask(String description) {
+    super(description);
+  }
+
+  /**
+   * Constructor for ToDo task, specifying completion status.
+   *
+   * @param description Description of task
+   * @param isDone      Boolean representing task completion status
+   */
+  public ToDoTask(String description, boolean isDone) {
+    super(description, isDone);
+  }
+
+  /**
+   * Gets Todo task formatted with type and status icon
+   *
+   * @return Task formatted as a string
+   */
+  @Override
+  public String toString() {
+    return "<T>" + super.toString();
+  }
+
+  /**
+   * Parses save file data into a To Do task instance.
+   *
+   * @param data line from save file
+   * @return Task instance
+   * @throws InvalidTaskDataException if data is not in the expected format
+   */
+  public static ToDoTask fromData(String data) throws InvalidTaskDataException {
```

Comment:
this could use the fromData method in Task so reduce the amount of duplicated code